openslides_backend/action/actions/meeting/import_.py

```diff
@@ -457,11 +507,11 @@ def create_write_requests(
         write_requests = []
         for collection in json_data:
             for entry in json_data[collection].values():
-                entry.pop("meta_new", None)
+                meta_new = entry.pop("meta_new", None)
```

Comment:
Only create a write request as EventType.Create. IMO we shouldn't change the user data on meeting import. We could expect that they are superior compared with the data of a import of old data.
This could be a difference to user import from client!

Reply:
Belonging to method replace_field_ids: We shouldn't overwrite the calculated fields user.meeting_ids, because it can't be filled correctly from an import. It should be calculated by the calculated field handler. Please check and write a test for this.